src/controllers/post.js

```diff
@@ -150,3 +150,117 @@ export const deletePost = async (req, res) => {
     sendMessageResponse(res, 500, 'Unable to delete post')
   }
 }
+
+export const createLike = async (req, res) => {
+  const postId = Number(req.params.id)
+
+  try {
+    const foundPost = await dbClient.post.findUnique({
+      where: { id: postId },
+      include: { user: true }
+    })
+    if (!foundPost) {
+      return sendMessageResponse(
+        res,
+        404,
+        'The post with the provided id does not exist'
+      )
+    }
+
+    const foundLike = await dbClient.like.findUnique({
+      where: {
+        userId_postId: {
+          userId: req.user.id,
+          postId
```

Comment:
Lets be explicit when adding the `postId` here to keep it consistent with `userId`

So instead let's do this:
```js
{
  userId: req.user.id,
  postId: postId
}
```